test/unit/dat-worker-pool_tests.rb

```diff
@@ -81,20 +69,21 @@ class InitTests < InitSetupTests
 
     should "build a runner" do
       exp = {
-        :num_workers  => @num_workers,
-        :queue        => @queue,
-        :worker_class => DefaultWorker,
-        :do_work_proc => @options[:do_work_proc]
+        :num_workers   => @num_workers,
+        :queue         => @queue,
+        :worker_class  => @worker_class,
+        :worker_params => @worker_params
       }
       assert_equal exp, @runner_spy.args
     end
 
     should "default its attributes" do
-      worker_pool = @worker_pool_class.new
+      worker_pool = @worker_pool_class.new(@worker_class)
       assert_instance_of NullLogger, worker_pool.logger
       assert_instance_of DatWorkerPool::DefaultQueue, worker_pool.queue
 
       assert_equal DEFAULT_NUM_WORKERS, @runner_spy.args[:num_workers]
+      assert_equal({}, @runner_spy.args[:worker_params])
```

Comment:
@jcredding like I mentioned above, I'd probably just let this default to `nil` if nothing is passed for params - nothing was passed after all.